frontend/src/components/atoms/MonComptePro/style.css

```diff
@@ -0,0 +1,3 @@
+.fr-connect-group a {
+  color: var(--text-action-blue-france);
```

Comment:
not needed, the link should be grey when visited. We should stick on the DSFR here.